New Live Testing capability for Device Passcode (#1358)
saucelabs/sauce-docs: docs/mobile-apps/live-testing/live-mobile-app-testing.md

| Device Language | Use the dropdown to select the device language. |
| Device Orientation | Use the dropdown to set the device orientation (Landscape or Portrait). |
| Proxy | Enable/disable the use of a proxy. Enter the **Hostname** and **Port** and then click **Update**. |

> | Device Passcode <br/><p><span className="sauceDBlue">ANDROID ONLY</span></p> | Enable/disable setting up device passcode on the device. If you are testing an Android Mobile app which requires a Passcode to be set on the device, this setting enables you to setup the passcode so your application can be launched. |

Reviewer: ```suggestion
| Device Passcode <br/><p><span className="sauceDBlue">Android Only</span></p> | Enable/disable the device passcode for Android apps. If your app requires a device passcode to launch, you can enable this setting to run your tests on a passcode-protected device. |
```
